pylutron/__init__.py

```diff
@@ -324,12 +325,25 @@ def send(self, op, cmd, integration_id, *args):
     self._conn.send(op + out_cmd)
 
   def load_xml_db(self):
```

Comment:
I think providing the cache should be handled at load_xml_db time since the top level object doesn't care.